lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1896,6 +1896,24 @@ static bool ParseIRGenArgs(IRGenOptions &Opts, ArgList &Args,
     Opts.EnableGlobalISel = true;
   }
 
+  // Default to disabling swift async extended frame info on linux.
+  if (Triple.getOS() == llvm::Triple::Linux) {
```

Comment:
Should this be `!Triple.getOS().isDarwin()`? I don't think the stack walkers anywhere else can cope with the extended frame info.

Reply:
Good point